azure-quantum/tests/unit/test_job.py

```diff
@@ -19,7 +19,7 @@
 from azure.quantum.optimization import Problem
 from azure.quantum.optimization.solvers import SimulatedAnnealing
 from azure.quantum import Job
-from tests.quantum_test_base import QuantumTestBase 
+from quantum_test_base import QuantumTestBase 
```

Comment:
I think this should be ..quantum_test_base, since the quantum_test_base is one level above unit folder. I get this import failure.

Reply:
Actually nvm, i see that you moved the quantum_base_file.py to unit directory. I think that is unnecessary. 
Here's my 2 c: 
1. I think this should be named common.py instead of quantum_test_base.py, just as a clarification of intent of this file. 
2. It should be outside of the unit folder. It may be used for other tests as well. Using  from ..quantum_test_base import QuantumTestBase should get around this.

Reply:
All this applies to the remaning comments as well.

Reply:
I would to get rid of the other test folder and only have unit tests that can be run manually, automated, live or in playback mode.
That's why I moved the file here as an initial step.
Makes sense?

I agree with renaming it to common.

Reply:
So we will eventually have a flat structure inside /tests where all the uts will be located?

Reply:
I am not opposed to this, I am just trying to plan for the changes I am adding now.

Reply:
Yes, we shouldn't need to distinguish the types of tests in different folders.
So I'll go ahead and merge it, but I'm open to suggestion that will come from your PR.